src/openms_gui/include/OpenMS/VISUAL/TOPPViewIdentificationViewBehavior.h

```diff
@@ -34,7 +34,7 @@
 
 #pragma once
 
-// #include <OpenMS/CHEMISTRY/NASequence.h>
+#include <OpenMS/CHEMISTRY/NASequence.h>
```

Comment:
actually this could go into the cpp and you can forward-declare `class NASequence` here!